Breeze.Sharp/NamingConvention.cs

```diff
@@ -85,6 +85,13 @@ public class NamingConvention : Internable {
       return clone;
     }
 
+    public NamingConvention WithServerClientNamespaceMapping(IDictionary<String, String> serverClientNamespaceMap) {
```

Comment:
I need another type of mapping used in https://github.com/enkodellc/blazorboilerplate/blob/development/src/Shared/BlazorBoilerplate.Shared/Services/ApiClient.cs